tensorflow/lite/micro/tools/requantize_flatbuffer.py

```diff
@@ -87,6 +87,8 @@
     schema_py_generated.BuiltinOperator.LEAKY_RELU,
     schema_py_generated.BuiltinOperator.LOGISTIC,
     schema_py_generated.BuiltinOperator.PAD
+    schema_py_generated.BuiltinOperator.STRIDED_SLICE,
+    schema_py_generated.BuiltinOperator.CONCATENATION,
```

Comment:
can I request that this list be ordered alphabetically? thanks!

Reply:
Good idea. Done